client/src/App.css

```diff
@@ -128,33 +100,26 @@ nav a.active {
     box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
 }
 
-
-
 /* buttons */
 
 .login-signin-buttons {
  display: flex;
  justify-content: center;
  gap: 20px;
-
 }
 
-
 button {
     padding: 15px 25px;
     background-color: rgb(219, 104, 104);
     border: none;
-    border-radius: 7px;
-    display: block;
+    border-radius: 7px;   
     width: 110px;
```

Comment:
What happens when you scale up the interface by 200%

https://www.accessguide.io/guide/resize-text-200